content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -32,21 +33,28 @@ An example of the request format:
     "params": [
         {}
     ]
+},
```

Comment:
Same thing applies here as the WebSocket version as commented above

Reply:
@mDuo13 It's been a while since I saw this PR, apologies for that.

The JSON-RPC contains both types of comma-separated examples, is that not what's expected?

Reply:
Is it better to keep only one of these two examples?